public/ending/ending.js

```diff
@@ -1,3 +1,15 @@
+import{getEndJson} from"./getQuizEnd.js";
+
+const urlParams=new URLSearchParams(window.location.search);
+const distParameter=urlParams.get("dist");
+const endIndex=distParameter.substring(distParameter.indexOf('-') + 1);
```

Comment:
```suggestion
const destParameter=urlParams.get("dest");
const endIndex=destParameter.substring(destParameter.indexOf('-') + 1);
```

destinationの略ならdestが正しそうです。